README.rst

```diff
@@ -36,10 +36,15 @@ Installation
 Create an ``ansible.cfg`` file in your project directory to tell
 Ansible where to install your roles (optionally, set the
 ``ANSIBLE_ROLES_PATH`` environment variable to do the same thing, or
-allow the roles to be installed into ``/etc/ansible/roles``) ::
+allow the roles to be installed into ``/etc/ansible/roles``).
+You should also enable ssh pipelining for performance, and might
+optionally want to enable ssh agent forwarding.::
 
     [defaults]
     roles_path = deployment/roles/
+    [ssh_connection]
+    pipelining = True
```

Comment:
At the bottom of this README, is a section named 'Optimizations', that includes some info on ssh pipelining. Wonder if it's worth deleting that section and moving the warning up here if appropriate.